sdk/core/Azure.Core/src/Shared/OperationInternalBase.cs

```diff
@@ -33,6 +36,7 @@ protected OperationInternalBase(ClientDiagnostics clientDiagnostics, string oper
         {
             _diagnostics = clientDiagnostics;
             _updateStatusScopeName = $"{operationTypeName}.UpdateStatus";
+            _waitForCompletionScopeName = $"{operationTypeName}.WaitForCompletion";
```

Comment:
Could we instead store a single private field `_operationTypeName` and append `.UpdateStatus`/`.WaitForCompletion` in the appropriate public methods?

Reply:
Oh, I see it was addressed!  Thanks!

Reply:
Should we use `{operationTypeName}.{(nameof(WaitForCompletionResponse)}` and `{operationTypeName}.{(nameof(WaitForCompletion)}` here?

Reply:
We can only add `{operationTypeName}.{(nameof(WaitForCompletionResponse)}`.
`WaitForCompletion` does not exist in `OperationInternalBase`.